yaksh/code_evaluator.py

```diff
@@ -108,9 +108,14 @@ def safe_evaluate(self, user_answer, test_case_data, file_paths=None):
 
         except TimeoutException:
             err = self.timeout_msg
+        except OSError:
+            msg = traceback.format_exc(limit=0)
+            err = "Error: {0}".format(msg)
         except Exception:
-            err = "Error: {0}".format(traceback.format_exc(limit=0))
-
+            exc_type, exc_value, exc_tb = sys.exc_info()
+            tb_list = traceback.format_exception(exc_type, exc_value, exc_tb)
+            del tb_list[1:3]
```

Comment:
I have added test cases for some exceptions and I have manually tested almost all built-in exceptions from interface also. It worked fine. 
lines which are removed from `tb_list` contains file path from which error occurred which can be removed.